app/views/courses/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="container-fluid">
+  <div class="row">
+      <div class="col-lg-8 col-md-6 col-sm-4"><h1>Listing Courses</h1></div>
+      <div class="col-lg-4 col-md-6 col-sm-8" style="text-align:right;"><h1><%= link_to "New Course", new_user_course_path(@user), :class => "btn"%></h1></div>
```

Comment:
Out of curiosity, why is the text-alight:right a style entry instead of inside an alight-right class or something like that? It just seems a bit inconsistent, epecially when this might be something we do frequently.

Reply:
I believe @mcshen worked on this bit